code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -1,111 +1,250 @@
+		) = 1,
+	))
+
+
+
+GLOBAL_LIST_INIT(common_loot, list( //common: basic items
+	list(//tools
+		/obj/item/screwdriver = 1,
+		/obj/item/wirecutters = 1,
+		/obj/item/wrench = 1,
+		/obj/item/crowbar = 1,
+		/obj/item/t_scanner = 1,
+		/obj/item/geiger_counter = 1,
+		/obj/item/analyzer = 1,
+		/obj/item/mop = 1,
+		/obj/item/reagent_containers/glass/bucket = 1,
+		/obj/item/toy/crayon/spraycan = 1,
+		) = 1,
+
+	list(//equipment
+		/obj/item/clothing/mask/gas = 1,
+		/obj/item/radio/headset = 1,
+		/obj/item/storage/backpack = 1,
+		/obj/item/clothing/shoes/sneakers/black = 1,
+		/obj/item/clothing/suit/hazardvest = 1,
+		/obj/item/clothing/suit/toggle/labcoat = 1,
+		/obj/item/clothing/under/color/grey = 1,
+		/obj/item/clothing/gloves/color/fyellow = 1,
+		/obj/effect/spawner/lootdrop/gloves = 1,
+		/obj/item/storage/wallet/random = 1,
+		/obj/item/clothing/glasses/science = 1,
+		/obj/item/clothing/glasses/meson = 1,
+		/obj/item/storage/belt/fannypack = 1,
+		) = 1,
+
+	list(//construction and crafting
+		/obj/item/stack/cable_coil = 1,
+		/obj/item/stock_parts/cell = 1,
+		/obj/item/stack/rods/twentyfive = 1,
+		/obj/item/stack/sheet/metal/twenty = 1,
+		/obj/item/stack/sheet/mineral/plasma = 1,
+
+		//assemblies
+		/obj/item/assembly/infra = 1,
+		/obj/item/assembly/signaler = 1,
+		/obj/item/assembly/mousetrap = 1,
+		/obj/item/assembly/prox_sensor = 1,
+		/obj/item/assembly/timer = 1,
+		/obj/item/assembly/igniter = 1,
+		/obj/item/assembly/health = 1,
+		/obj/item/stack/packageWrap = 1,
+		) = 1,
+
+	list(//medical and chemicals
+		/obj/item/reagent_containers/pill/floorpill = 1,
+		/obj/item/storage/fancy/cigarettes/dromedaryco = 1,
+		/obj/item/grenade/chem_grenade/cleaner = 1,
+		/obj/item/storage/box/matches = 1,
+		/obj/item/reagent_containers/syringe = 1,
+		/obj/item/reagent_containers/glass/beaker = 1,
+		/obj/item/reagent_containers/glass/rag = 1,
+		) = 1,
+
+	list(//food
+		/obj/item/reagent_containers/food/drinks/beer = 1,
+		/obj/item/reagent_containers/food/drinks/coffee = 1,
+		) = 1,
+
+	list(//misc
+		/obj/item/radio/off = 1,
+		/obj/item/extinguisher = 1,
+		/obj/item/tank/internals/emergency_oxygen = 1,
+		/obj/item/bodybag = 1,
+		/obj/item/grenade/smokebomb = 1,
+		/obj/item/stack/spacecash/c10 = 1,
+
+		//light sources
+		/obj/item/flashlight = 1,
+		/obj/effect/spawner/lootdrop/glowstick = 1,
+		/obj/item/clothing/head/hardhat/red = 1,
+		/obj/item/flashlight/flare = 1,
+		) = 1,
+
+	list(//pump-up
+		/obj/item/reagent_containers/hypospray/medipen/pumpup = 1,
+		) = 1,
+	))
+
+
+
+GLOBAL_LIST_INIT(uncommon_loot, list(//uncommon: useful items
+	list(//tools
+		/obj/item/weldingtool/mini = 1,
+		/obj/item/multitool = 1,
+		/obj/item/hatchet = 1,
+		/obj/item/roller = 1,
+		/obj/item/restraints/legcuffs/bola = 1,
+		/obj/item/restraints/handcuffs/cable = 1,
+		/obj/item/twohanded/spear = 1,
+		/obj/item/shield/riot/buckler = 1,
+		/obj/item/grenade/iedcasing = 1,
+		/obj/item/melee/baton/cattleprod = 1,
+		) = 1,
```

Comment:
these are all pretty toasty of items, have you tested how frequently these spawn?

Reply:
Each item in your marked section currently has a 1/600 probability of selection. (1/10 to select uncommon, 1/6 to select that tools list, 1/10 to select any given sub-item). For metastation, with ~180 pieces of total loot spawned, an average of 3 from this section, and 0.3 of each individual item will spawn, and an average of 18 items from the uncommon category overall.

Reply:
The three defines from lines 239-242 control the spawn rates of each main category. If you feel that the loot is too strong, I could move the uncommon selection rate down from 1/10 to 3/100, or whatever, it's fairly flexible and I'm not attached to any of the spawn rates.

Reply:
~3 is a not entirely unreasonable number for that value. It can always be tweaked going forward.